lib/iris/tests/stock/_stock_2d_latlons.py

```diff
@@ -26,7 +26,7 @@
 import numpy.ma as ma
 
 from iris.analysis.cartography import unrotate_pole
-from iris.cube import Cube
+from iris.cube import Cube, CubeList
```

Comment:
We simplified `iris.util.mask_cube` (which had pretty much the same masking operations in) but not this.  I don't have a problem with keeping this as it is because it's only to generate test data, which means it won't be used on large data sets so it won't eat up lots of processing time slicing up cubes and masking them.